pkgs/misc/betterfox/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "betterfox";
+  version = "119.0";
+
+  src = fetchFromGitHub {
+    owner = "yokoffing";
+    repo = "Betterfox";
+    rev = version;
```

Consider using `finalAttrs`:

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
  pname = "betterfox";
  version = "119.0";

  src = fetchFromGitHub {
    owner = "yokoffing";
    repo = "Betterfox";
    rev = finalAttrs.version;
```